### ISSUE TITLE
Nemo does not show thumbnails of unzipped files downloaded from Outlook

### ISSUE THREAD

**Issue by Angelca6** [linuxmint/nemo#3427]
### Distribution

Mint Cinnamon 21.3

### Package version

6.0.2

### Frequency

Always

### Bug description

Nemo has a bug that no one has noticed, when you download several files from Outlook email, they are downloaded compressed. When you unzip them, Nemo does not show the thumbnails. I was able to reproduce the error on several computers.  As additional information, the mini strips of the decompressed files that were downloaded from Outlook recently can be viewed later. 3 hours or so, this problem does not exist in Linux Mint cinnamon 20.3. 
The first image shows the view with thumbnails enabled, the second image shows the view with thumbnails disabled.
![Nemo](https://github.com/linuxmint/nemo/assets/174410992/e92adab7-60dd-406a-8c86-7cbe8bc26155)
![Nemo 2](https://github.com/linuxmint/nemo/assets/174410992/10879cbb-8ce1-46c0-8530-e2d2c89f4b6e)

### Steps to reproduce

1. Open an Outlook email
2. Search for an email with several attachments
3. Download all files at once
Files will be downloaded as a zipped folder
4. Right click on the compressed folder and use the extract here option
5. Open the folder and you will notice that the miniatures are not shown, instead you see a kind of clock icon.

### Expected behavior

The thumbnails of the files should be displayed correctly to be able to see, for example, the preview of the PDFs or the images in addition to being able to differentiate one type of file from another but it is not possible because a kind of clock appears for all the files. file types alike.

### Additional information

As additional information, the thumbnails can be displayed correctly after 3 hours or so and this problem does not occur when I test the distributions on a USB Live. The problem started from linux Mint 21.1

**Comment by Angelca6** [comment]
I also forgot to indicate that if the file is a text document, a presentation or a spreadsheet, when editing it and saving the change, the thumbnails can be seen immediately.
The first image shows the view of the uncompressed files with the thumbnails activated, the second image (the thumbnails are still activated) shows that when editing the text and saving you can already see the icons of the modified text files, finally The third image shows the view of the files with the thumbnails disabled.
![Nemo 3](https://github.com/linuxmint/nemo/assets/174410992/08538f22-14e4-4a9a-a4d4-df638eeb7c2e)
![Nemo 4](https://github.com/linuxmint/nemo/assets/174410992/17a439a5-3d8f-4474-89eb-3b6c0fc0146d)
![Nemo 5](https://github.com/linuxmint/nemo/assets/174410992/af888f45-f3fc-4051-9c6d-7fc79040b853)

**Comment by archisman-panigrahi** [comment]
Is this bug present in Mint 22?

**Comment by Angelca6** [comment]
Estaba en la beta de Mint 22

**Comment by Angelca6** [comment]
Yes, this error is also in the Linux Mint 22 version, I was able to reproduce the error on 10 different machines, those in my house and those of my coworkers on whom I installed Linux Mint cinnamon, for work unfortunately I am forced to use outlook and my colleagues and also for this reason this problem affects us, as a solution we are currently using thunar as a file manager instead of nemo

**Comment by archisman-panigrahi** [comment]
> when you download several files from Outlook email, they are downloaded compressed. When you unzip them, Nemo does not show the thumbnails

Is outlook relevant here? When you download any zip file (not just from outlook) and unzip it, is the error present? Also, what happens when you refresh the folder? (You can add a refresh button on the menu)

**Comment by Jeremy7701** [comment]
F5 is a refresh key on nemo - it is occasional needed to get a thumbnail regenerated on EPUB files.

**Comment by Angelca6** [comment]
Outlook email is important because if we want to bring more users to Linux Mint, many of these new users use this email for their work. In my case, unfortunately, I am forced to use Outlook at the university where I work as a teacher. I installed Linix Mint cinnamon for eight friends at work, but upon seeing this error in Nemo where the thumbnails do not appear when unzipping the files downloaded from Outlook, they wanted to return to Windows. To prevent them from returning to Windows, I installed thunar as a file manager to that they can work without problems which was a momentary solution.

**Comment by archisman-panigrahi** [comment]
By outlook email do you mean you use outlook in a browser? If so, I can try to replicate this. My university uses outlook too.

**Comment by Angelca6** [comment]
That's how I use Outlook in the browser, look for an email that has several attachments, when you download them all at once you will get a compressed file, unzip it with right click on the extract option here, if the thumbnail view is activated in Nemo you will see that the Thumbnail icons are not displayed correctly. If you deactivate the thumbnail views, you will see the icons but not the preview of images or pdf. I invite you to try to replicate it and comment on it.

**Comment by Xelveki** [comment]
We are also having this issue with photos downloaded from iCloud. If you download multiple files, it puts them into an archive and sends you the archive. After extracting from the archive, the photos do not have a thumbnail.

**Comment by dollyp** [comment]
Mint 22.1 Cinnamon 6.4.8
Outlook and email attachments are not themselves the issue. I scanned some photos on a Win10 machine (only thing I use it for) then copied the .jpegs over to Nemo via a USB stick and the same problem. Viewing in Pix then back to Nemo made the thumbnails visible. There's something happening in the transfer of files from NTFS to ext4 and it may affect all sorts of files not just jpegs and email attachments.

**Comment by zenvoid** [comment]
It happens to me when I copy files from my camera. They won't have previews and the clock icon is stuck forever. I made some tests:
- When I type in a terminal "cp file.jpg file2.jpg", the new copy will have preview.
- If I then type "cp file2.jpg file.jpg" (override the original file with a copy of itself), it's fixed and both copies will have previews.
- If I rename the file instead of copying, "mv file.jpg file2.jpg", the clock icon remains and there is no preview.
So moving or renaming doesn't fix the preview but copying does.

**Comment by Jeremy7701** [comment]
Move / rename doesn't touch the data in a file; obviously a copy creates a new file (and its contents).
